modules/@apostrophecms/doc-type/index.js

```diff
@@ -91,6 +91,7 @@ module.exports = {
     self.composeSchema();
     self.apos.doc.setManager(self.name, self);
     self.enableBrowserData();
+    self.addContextMenu();
```

Comment:
Not sure this `doc-type` is the best place to add the "share draft" option to the context menu.. or is it?